src/components/app/Home.css

```diff
@@ -188,6 +188,29 @@
   transition: opacity 300ms ease-out, transform 300ms ease-out;
 }
 
+.homeSectionUploadFiles {
+    padding: 0.5em;
+    margin: 15px 0;
+    display: block;
+    border: 2px solid var(--blue-50);
+    color: #fff;
+    background-color: var(--blue-50);
+    transition: background-color 300ms;
+    font-weight: bold;
+    font-size: 16px;
+    text-align: center;
+    cursor: pointer;
+}
+
+.homeSectionUploadFiles:hover {
+    color: var(--blue-50);
+    background-color: #fff;
+}
+
+input[type="file"] {
+    display: none;
+}
```

Comment:
You can't use `display: none` because then we can't reach the input using the keyboard, which is very unfortunate.

See [this article](https://tympanus.net/codrops/2015/09/15/styling-customizing-file-inputs-smart-way/) for a possible solution.